addon-test-support/setup-window-mock.js

```diff
@@ -0,0 +1,11 @@
+import { reset } from 'ember-window-mock';
+
+//
+// Used to reset window-mock for a test.
+//
+// NOTE: the `hooks = self` is for mocha support
+//
+export default function setupWindowMock(hooks = self) {
+  hooks.beforeEach(reset);
+  hooks.afterEach(reset);
```

Comment:
Not sure, do we really need to run it before *and* after each test? Shouldn't `beforeEach` be sufficient?

Reply:
I've updated the PR to only reset in the beforeEach hook